collections/_article/godot-xr-community-nov-2024.md

```diff
@@ -0,0 +1,110 @@
+
+There were far more great entries, so be sure to [check out the jam page for more](https://itch.io/jam/godot-xr-game-jam/entries)!
+
+### Godot XR Game Jam #2
+
+The [second Godot XR Game Jam](https://itch.io/jam/godot-xr-game-jam-july-2024) was held from 8 July until 15 July 2024.
+We decided to give participants a full week to complete their game.
+The theme this jam was Relaxation. There were a total of 26 entries and this time we scored them on the average points in all categories.
+
+The #1 spot went to [Who's a good boy](https://itch.io/jam/godot-xr-game-jam-july-2024/rate/2821459) with 4.042 points.
+This cute little game has you play fetch with a dog in a park.
+It features some excellent interactions both from the player's and dog's points of view.
+
+The #2 spot went to [Constellation Coach VR](https://itch.io/jam/godot-xr-game-jam-july-2024/rate/2830514) with 3.974 points.
+This game has you find constellations in the night sky by holding up a picture of the constellation and aligning it with the relevant stars.
+Excellent haptic feedback lets you know when you've gotten close, and successfully aligning the constellation lights it up.
+
+The #3 spot went to [Summer Pottery VR](https://itch.io/jam/godot-xr-game-jam-july-2024/rate/2830121) with 3.938 points.
+This game lets you do some clay pottery sculpting.
+It's a great and very well-executed use of physically interacting with the clay to shape and paint your creation.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/0IxvWaj8iUo?si=em-D99NacStXyIYA" title="5 VR Games Made in Godot To Inspire You" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+Again, many of the other entries were of great quality and we highly recommend [checking them out](https://itch.io/jam/godot-xr-game-jam-july-2024/entries).
+
+### Godot XR Game Jam #3
+
+The 3rd game jam which will be held in the week starting 10 February 2025. [You can read all about it and sign up here](https://itch.io/jam/godot-xr-game-jam-feb-2025).
+
+## Universal Godot VR plugin
+
+Inspired by the Universal Unreal Engine VR Mod, community member TeddyBear082 has created a mod to inject VR capabilities into existing 3D games made with Godot.
+
+TeddyBear082 previously made a name for himself with the excellent VR port of the popular game Cruelty Squad and has used the knowledge gained with that and similar projects to create a more universal mod.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Hb9BDcCZHCY?si=Zr2Jw3MV7cY00yEE" title="
+
+
+0:03 / 2:01
+
+UGVR Demo" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

Comment:
Probably this `<iframe>` isn't meant to be split across so many lines like that? I'm guessing this is intended to be:

```suggestion
<iframe width="560" height="315" src="https://www.youtube.com/embed/Hb9BDcCZHCY?si=Zr2Jw3MV7cY00yEE" title="UGVR Demo" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

Reply:
Oops, that was a copy paste mistake :)